### DISCUSSION TITLE
Mantine V7 Browser Support

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#7117] [Q&A] [answered]
Before ask any questions, thank you for creating such a great library!

I'd like to know about mantine's specific browser support range! 

In the document, it is as follows. (Chrome - 108+)

https://mantine.dev/about/#browser-support

In fact, it works normally in lower browsers as well. (working Chrome - 84+)

It seems to support even lower versions than the actual browser version shown.

May I know how low the version supports Safari and Firefox?

I will move the user who accessed the lower version to the guide page.

**Comment by rtivital** [comment, marked as answer]
Thanks for kind words!

The browser support link you've shared includes a list of CSS features used by all/some components. Some of these features are available in older browsers.

**Reply by ryuhangyeong** [reply]
@rtivital 

I'm putting this together because I think it may be helpful to someone.

- The version written in the document is a stable version, and Mantine can actually be used in older browsers of a certain version or higher.
- Most JavaScript codes can be responded to simply by adding a polyfill.
- Mantine actively uses the latest CSS for efficient code writing and performance. In my personal opinion, the thing that has the biggest impact on browser compatibility is CSS, and it would be better if there was a way to deal with it, but there isn't.
  - However, since the majority of general users are using the latest browsers, I do not think it is worth spending maintenance time to support older browsers for a small number of people.
- Unless absolutely necessary, it is reasonable to redirect to a page that instructs you to update your browser when accessed using a browser that is lower than a certain version, rather than wasting time responding to an outdated browser.
